packages/@react-types/buttongroup/src/index.d.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+
+import {DOMProps, Orientation, StyleProps} from '@react-types/shared';
+import {ReactNode} from 'react';
+
+export interface SpectrumButtonGroupProps extends DOMProps, StyleProps {
+  /** Whether the ButtonGroup is disabled. */
+  isDisabled?: boolean,
+  /**
+   * The axis the ButtonGroup should align with.
+   * @default "horizontal"
+   */
+  orientation?: Orientation,
```

Comment:
Should it be mentioned that if vertical is used, no reflow will happen, it'll just always be vertical. But if horizontal is used, then reflow to vertical will happen?

Reply:
I think this should be documented in the docs, I'll say something like "See the docs for reflow behavior".

Reply:
will be easier to link to it from here